cypress/integration/finish_mama_project/social.spec.js

```diff
@@ -0,0 +1,191 @@
+///<reference types='cypress' />
+import { getRandomNumber } from '/cypress/support/utils.js';
+import { login } from '/cypress/support/shared.js';
+import meUser from '/cypress/fixtures/me-user.json';
+import { setJwtToken } from '/cypress/support/utils';
+
+function selectFirstArticle() {
+  waitForArticlesList();
+  cy.get('@articleList')
+    .find('.article-preview')
+    // .should('have.length', 12)
+    .should('have.length.greaterThan', 2)
+    .eq(0)
+    .as('firstArticle');
+}
+function waitForArticlesList() {
+  cy.get('@articleList')
+    .contains("div.article-preview", "Loading")
```

Comment:
- [ ] fix double quotes to single
- [ ] don't use none semantic tags as selectors `.article-preview`

Reply:
Is it right?
.contains('.article-preview', 'Loading')